src/simulation/LoadGenerator.cpp

```diff
@@ -988,10 +1040,21 @@ std::pair<LoadGenerator::TestAccountPtr, TransactionFramePtr>
 LoadGenerator::creationTransaction(uint64_t startAccount, uint64_t numItems,
                                    uint32_t ledgerNum)
 {
-    TestAccountPtr sourceAcc =
-        mInitialAccountsCreated
-            ? findAccount(getNextAvailableAccount(), ledgerNum)
-            : mRoot;
+    TestAccountPtr sourceAcc = mRoot;
+    if (mInitialAccountsCreated)
+    {
+        std::optional<uint64_t> accountId = getNextAvailableAccount(ledgerNum);
+        if (!accountId.has_value())
+        {
+            // This really shouldn't happen and indicates a misconfiguration
```

Comment:
Given that this shouldn't happen, should we keep the code as-is and allow it to crash? Otherwise, the exception suggests that this can be remedied by adjusting parameters (but this just hides a bug in sheduleLoadGeneration)

Reply:
I reverted this. This check isn't necessary anymore after getting rid of the `std::optional`